tests/acceptance/features/bootstrap/OcmContext.php

```diff
@@ -74,24 +103,55 @@ public function userGeneratesInvitation(string $user, $email = null, $descriptio
 		);
 		$this->featureContext->setResponse($response);
 		$this->invitationToken = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())['token'];
+		return $response;
 	}
 
 	/**
-	 * @When :user accepts invitation
+	 * @Given :user has created the federation share invitation
 	 *
 	 * @param string $user
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function userAcceptsInvitation(string $user): void {
+	public function userHasCreatedInvitation(string $user): void {
+		$response = $this->createInvitation($user);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200, '', $response);
+	}
+
+	/**
+	 * @When :user accepts the last federation share invitation
+	 * @When :user tries to accept the last federation share invitation
+	 *
+	 * @param string $user
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public function acceptInvitation(string $user): ResponseInterface {
```

Comment:
let's have separate helper and stepDef methods
```suggestion
	/**
	 * @param string $user
	 *
	 * @return ResponseInterface
	 * @throws GuzzleException
	 */
	public function acceptInvitation(string $user): ResponseInterface {
	...
	}
	
	 /**
	 * @When :user accepts the last federation share invitation
	 * @When :user tries to accept the last federation share invitation
	 *
	 * @param string $user
	 *
	 * @return void
	 * @throws GuzzleException
	 */
	public function userAcceptsTheLastFederationShareInvitation(string $user): void {
	        $this->featureContext->setResponse($this->acceptInvitation($user));
	}
```